content/03.methods.md

```diff
@@ -388,6 +388,10 @@ Additional details can be found in the analysis [notebook](https://alexslemonade
 6. Non-MB and non-ATRT embryonal tumors with <i>CIC-NUTM1</i> or other <i>CIC</i> fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with <i>CIC</i> alteration).
 7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as CNS Embryonal, NOS (CNS Embryonal tumor, not otherwise specified).
 
+Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned based on the primary site of the tumor.
```

Comment:
Following back to original ticket (https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/805), this looks like the appropriate citation:

```suggestion
Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned based on the primary site of the tumor [@doi:10.1007/978-3-319-33432-5_20].
```